sim/warrior/sunder_armor.go

```diff
@@ -69,6 +72,7 @@ func (warrior *Warrior) newSunderArmorSpell(isDevastateEffect bool) *core.Spell
 			if warrior.SunderArmorAura.GetStacks() == 5 {
```

Comment:
Perhaps this conditional should be removed since Devastate and Sunder Armor each do their threat independently of each other now.